src/System Application/App/Extension Management/src/ExtensionMarketplace.Page.al

```diff
@@ -1,116 +0,0 @@
-#if not CLEAN24
-// ------------------------------------------------------------------------------------------------
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-// ------------------------------------------------------------------------------------------------
-
-namespace System.Apps;
-
-using System;
-using System.Utilities;
-using System.Integration;
-
-/// <summary>
-/// Shows the Extension Marketplace.
-/// </summary>
-page 2502 "Extension Marketplace"
-{
-    Caption = 'Extension Marketplace';
-    AdditionalSearchTerms = 'app,add-in,customize,plug-in,appsource';
-    PageType = Card;
-    ApplicationArea = All;
-    UsageCategory = Administration;
-    Editable = false;
-    ObsoleteState = Pending;
-    ObsoleteReason = 'This page will be obsoleted. Microsoft AppSource apps feature will replace the Extension Marketplace.';
-    ObsoleteTag = '24.0';
```

Comment:
ehh... it seems like we're not respecting the obsoletion period here. If something was obsoleted in v24 we should keep it functional until v26 at least and remove in v27 at the earliest, and since i see preprocessor symbols CLEAN24 added everywhere that process should be automatic. 
Removing it earlier would be seen as an unexpected breaking change.

Reply:
Yeah, I would agree here.
In order to be introduced now, we could probably hide old one

Reply:
You're right we were too eager to remove it with the deprecation from the app source team. Left the page here but removed the functionality. Should not break anyone except from a bad experience if they try to use it (blank page with only one new action).